metadata.rb

```diff
@@ -1,18 +1,17 @@
-name             'pdns'
+name             'chef-pdns'
```

Comment:
Why did this change? Our cookbook and supermarket name is pdns. If you uploaded this now it would create chef-pdns instead of pdns.